README.md

```diff
@@ -25,8 +25,8 @@ Add the entries below to the global `~/.gradle/gradle.properties` file at `$HOME
 ```bash
 
 # required for accessing to artifactory
-artifactoryUsername=###
-artifactoryPassword=###
+repositoryUsername=###
```

Comment:
I wonder if it would be better to change the files that use these properties to use the names shown in the README file as `artifactoryUsername` and `artifactoryPassword` are the property names used in the SDK project and other examples we've created, such as the [offline example](https://github.com/tomtom-internal/bue-cpm-android-offline-navigation) I maintain.

Reply:
Done. Please check 3eba518e6a6c539d8c2538fcae634c8c7af6385d . Thanks!